### ISSUE TITLE
'node_modules/@fastify/busboy/lib/utils.js': TypeError: Value is not an object: undefined

### ISSUE THREAD

**Issue by Tom3652** [fastify/busboy#100]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the bug has not already been reported

### Fastify version

See in firebase-admin SDK (messaging)

### Plugin version

See in firebase-admin SDK (messaging)

### Node.js version

MongoDB Atlas Trigger : JS SDK Version: 3.18.0

### Operating system

Linux

### Operating system version (i.e. 20.04, 11.3, 10)

No idea sorry

### Description

I don't know NodeJS at all sorry, but i am using the `firebase_admin` package that is actually using your package as a dependency. 

I have no clue about it but since they don't answer to my [issue](https://github.com/firebase/firebase-admin-node/issues/1902) there.
I am making an issue here so perhaps you could tell me what is happening : 

```
'http' module: FunctionError: failed to execute source for 'node_modules/@fastify/busboy/lib/main.js': FunctionError: failed to execute source for 'node_modules/@fastify/busboy/deps/dicer/lib/Dicer.js': FunctionError: failed to execute source for 'node_modules/@fastify/busboy/deps/dicer/lib/HeaderParser.js': FunctionError: failed to execute source for 'node_modules/@fastify/busboy/lib/utils.js': TypeError: Value is not an object: undefined
	at node_modules/@fastify/busboy/lib/utils.js:23:27(322)

	at require (native)
	at node_modules/@fastify/busboy/deps/dicer/lib/HeaderParser.js:15:24(29)

	at require (native)
	at node_modules/@fastify/busboy/deps/dicer/lib/Dicer.js:19:28(39)

	at require (native)
	at node_modules/@fastify/busboy/lib/main.js:28:21(44)
```

I must specify that this error is not impacting my service, but i am having multiple errors in my Atlas trigger because of this. 

### Steps to Reproduce

You will find all the detail in the mentioned issue. 

### Expected Behavior

This error could be more explicit (with the wrong value logged for example).

I would especially really appreciate you tell me what can be wrong here or answer directly in the Firebase repo with what's wrong in their code please.

**Comment by kibertoad** [comment]
I'll take a look, thanks!

**Comment by Eomm** [comment]
Not sure if it is related, but when I see dicer, I recall it:
https://security.snyk.io/package/npm/dicer

**Comment by kibertoad** [comment]
@Eomm It used to be the case, but @Uzlopak fixed security vulnerabilities in our fork.

**Comment by lahirumaramba** [comment]
@kibertoad IIUC it looks like `HeaderParser` uses `getLimit()` from `libs/utils`... do you think we can move`getLimit` out of `utils` or move to `Dicer`? Happy to submit a PR if this makes sense to you. Thanks!
https://github.com/fastify/busboy/blob/05c990c4f42c00834ed0c717ea29c564d379cf0c/lib/utils.js#L177

**Comment by Tom3652** [comment]
Hello guys, 
I am sorry to ask but is that meaning the issue is solved ? I can check it ? 
Thanks for your work

**Comment by Uzlopak** [comment]
Yes, we released 1.2.0 today. Please Check
